build/ci/tools/osuosl/publish.sh

```diff
@@ -72,33 +80,20 @@ echo $PUBLISH_URL > $ARTIFACTS_DIR/env/publish_url.env
 cat $ARTIFACTS_DIR/env/publish_url.env
 
 # Delete old files
-if [ "$BUILD_MODE" == "nightly_build" ]; then 
+if [ "$BUILD_MODE" == "nightly_build" ]; then
     echo "Delete old MuseScoreNightly files"
 
-    if [ "$OS" == "linux" ]; then 
-        ssh -i $SSH_KEY musescore-nightlies@ftp-osl.osuosl.org "cd ~/ftp/$FTP_PATH; ls MuseScoreNightly*.AppImage       -t | tail -n +41 | xargs rm -f"
-        ssh -i $SSH_KEY musescore-nightlies@ftp-osl.osuosl.org "cd ~/ftp/$FTP_PATH; ls MuseScoreNightly*.AppImage.zsync -t | tail -n +41 | xargs rm -f"
-    else 
-        ssh -i $SSH_KEY musescore-nightlies@ftp-osl.osuosl.org "cd ~/ftp/$FTP_PATH; ls MuseScoreNightly* -t | tail -n +41 | xargs rm -f"
-    fi
+    ssh -i $SSH_KEY musescore-nightlies@ftp-osl.osuosl.org "cd ~/ftp/$FTP_PATH; ls MuseScoreNightly* -t | tail -n +42 | xargs rm -f"
 fi
 
 # Create link to latest
-if [ "$BUILD_MODE" == "nightly_build" ]; then 
+if [ "$BUILD_MODE" == "nightly_build" ]; then
 
     echo "Create link to latest"
 
     sleep 1m # For fine sort order on FTP
```

Comment:
I looked, it is still not sorted correctly, all 4 files have the same time. I think I need to try to set 2 minutes.
If that doesn't work, we might need to call the trigger, then delay, create links, and call the trigger again.

Reply:
@igorkorsukov, did you check via SSH or just looking at the [webpage](https://ftp.osuosl.org/pub/musescore-nightlies/linux/3x/nightly/?C=M;O=D)? The files have the same timestamp on the webpage (to the minute), but the sorting is done on the server rather than through JavaScript. Notice that:

- If you reverse the sort order on the webpage then the list reverses.
- If you switch to ordering by name then the files are ordered differently.

 So it appears that the server has access to the full timestamp (to the milli/micro second) rather than just to the nearest minute.

Regarding both zsync files appearing above both AppImages, I think that the system is using the timestamp of the original file rather than the symlink. Using hard links won't help here, unfortunately, but I could replace the symlink with a physical copy if the order matters more than disk space?

Reply:
I was looking at a web page.
If the link uses the timestamp of the file, then I suggest keep it as it is, you can even remove `sleep 1m`.
Let's see if someone complains